autogen.pl

```diff
@@ -1509,6 +1519,10 @@ sub in_tarball {
     }
     push(@subdirs, "3rd-party/openpmix/");
     $m4 .= "m4_define([package_pmix], [1])\n";
+
+    # Grab the unique configure options from each of the 3rd party packages
+    system("./config/extract-3rd-party-configure.pl -p \"3rd-party/openpmix/\" -n \"PMIx\" -m > config/auto-pmix.m4");
```

Comment:
don't forget to check and handle the return code.  We should abort configure if this fails.

Reply:
Will be fixed in the next push.